src/wp-includes/class-wp-script-modules.php

```diff
@@ -275,6 +284,34 @@ public function add_hooks() {
 		add_action( 'admin_print_footer_scripts', array( $this, 'print_a11y_script_module_html' ), 20 );
 	}
 
+	/**
+	 * Gets the highest fetch priority for the provided script IDs.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param string[] $ids Script module IDs.
+	 * @return string Highest fetch priority for the provided script module IDs.
+	 */
+	private function get_highest_fetchpriority( array $ids ): string {
+		static $priorities = array(
+			'low',
+			'auto',
+			'high',
+		);
+
+		$highest_priority_index = 0;
+		foreach ( $ids as $id ) {
```

Comment:
There's an opportunity here to break out of iteration once the max value is found with something like:

```suggestion
		foreach ( $ids as $id ) {
			if ( $highest_priority_index > 1) {
				break;
			}
```

Reply:
How about 546e7a78e302c66fa962eff5a1e1185cb6c3ca5d?

Reply:
Looks good.